contracts/ConvictionVoting.sol

```diff
@@ -225,22 +244,22 @@ contract ConvictionVoting is DisputableAragonApp, TokenManagerHook {
      * @param _proposalId Proposal id
      * @param _amount Amount of tokens withdrawn
      */
-    function withdrawFromProposal(uint256 _proposalId, uint256 _amount) external isInitialized() proposalExists(_proposalId) {
+    function withdrawFromProposal(uint256 _proposalId, uint256 _amount) external notPaused isInitialized proposalExists(_proposalId) {
         _withdrawFromProposal(_proposalId, _amount, msg.sender);
     }
 
     /**
      * @notice Withdraw all `(self.stakeToken(): address).symbol(): string` tokens previously staked on proposal #`_proposalId`
      * @param _proposalId Proposal id
      */
-    function withdrawAllFromProposal(uint256 _proposalId) external isInitialized() proposalExists(_proposalId) {
+    function withdrawAllFromProposal(uint256 _proposalId) external notPaused isInitialized proposalExists(_proposalId) {
         _withdrawFromProposal(_proposalId, proposals[_proposalId].voterStake[msg.sender], msg.sender);
     }
 
     /**
      * @notice Withdraw all callers stake from inactive proposals
      */
-    function withdrawFromInactiveProposals() external isInitialized() {
+    function withdrawFromInactiveProposals() external notPaused isInitialized {
```

Comment:
That is fair, I'll have a think. We also let users withdraw from active proposals through transfers btw.

Reply:
I think it would be a good idea to allow withdrawals. I've realised that to pause the accrual of conviction would be pretty complex. We would have to store the block at which each proposal is created and whenever the contract is paused we would have to store the block at which it is paused, then when accruing conviction we would have to check if a pause happened after the block at which the proposal was created and add the number of paused blocks. We would have to store every pause period and apply whichever's necessary for a specific proposal.